.github/workflows/ci.yml

```diff
@@ -1,7 +1,4 @@
-on:
-  pull_request:
-    branches:
-      - master
```

Comment:
not related to this PR, but should master be switched to main in this repo, like what we've done in other repos?